.htaccess.dist

```diff
@@ -4,33 +4,53 @@
 <IfModule mod_rewrite.c>
     RewriteEngine On
 
+    ####
     # Certain hosts may require the following line.
     # If vanilla is in a subfolder then you need to specify it after the /.
     # (ex. You put Vanilla in /forum so change the next line to: RewriteBase /forum)
-    # RewriteBase /
+    ####
+    #RewriteBase /
 
-    RewriteCond %{QUERY_STRING} ^p=/?([^&]+)(&([^?]+))?$
-    RewriteRule ^index\.php %1?%3 [E=X_REWRITE:1,L]
+    ####
+    # Deny access to certain directories that SHOULD not be exposed.
+    ####
+    RewriteRule (^|/)\.git - [L,R=403]
+    #RewriteRule ^build/ - [L,R=403] # Already covered by .htaccess
```

Comment:
In what scenario is someone uploading a `build` or `tests` folder that we'd need to account for it?